### DISCUSSION TITLE
Add userID or systemID field to the Contact object

### DISCUSSION THREAD

**Discussion by dss70** [netbox-community/netbox#8055] [Ideas]
Is it possible to add userID or systemID field to the Contact object? Users usually identified by a unique id of an external system.

**Comment by jeremystretch** [comment]
It would probably be best to use a custom field for this, as we can't assume what such an ID might look like (its data type, format, etc.).

**Comment by dss70** [comment]
In my opinion this would be a simple text field which could contain any kind of alphanumeric ID.
